charts/sps-sample/values.yaml

```
@@ -58,6 +58,22 @@ refarch-gateway:
     - name: SPRING_CLOUD_GATEWAY_ROUTES_1_FILTERS_1
       value: RemoveResponseHeader=WWW-Authenticate
 
+    # KEYCLOAK
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)